projects/plugins/boost/app/modules/optimizations/page-cache/Page_Cache.php

```diff
@@ -69,6 +69,10 @@ public function handle_module_status_updated( $module_slug, $status ) {
 		if ( in_array( $module_slug, $slugs, true ) ) {
 			$this->invalidate_cache();
 		}
+
+		if ( $module_slug === 'page_cache' && ! $status ) {
+			Page_Cache_Setup::delete_advanced_cache();
+		}
```

Comment:
This works nicely! However, as uncle Bob said, one function should do a single thing, and this function seems to be doing two things now. 

I suggest we split the functionality in two different functions. Something like:
- `clear_cache_on_output_changing_module_toggle()`
- `delete_advanced_cache`

Reply:
Good idea, done!